client/my-sites/email/paths.js

```diff
@@ -84,17 +100,36 @@ export function emailManagementForwarding( siteName, domainName, relativeTo = nu
 	return emailManagementEdit( siteName, domainName, 'forwarding', relativeTo );
 }
 
-export function emailManagementEdit( siteName, domainName, slug, relativeTo = null ) {
+export function emailManagementEdit(
+	siteName,
+	domainName,
+	slug,
+	relativeTo = null,
+	urlParameters = {}
+) {
 	slug = slug || 'manage';
 
 	// Encodes only real domain names and not parameter placeholders
-	if ( ! startsWith( domainName, ':' ) ) {
+	if ( domainName && ! String( domainName ).startsWith( ':' ) ) {
```

Comment:
Can `domainName` be from a type other than `string`? if no, then the following should work:

```suggestion
	if ( domainName && ! domainName.startsWith( ':' ) ) {
```

Reply:
I wasn't sure, TBH, so I went for a more defensive approach.